File: Source/frontend/frontend.Service/ICargoService.cs
using frontend.Domain;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace frontend.Service
{
    public interface ICargoService
    {
        List<Cargo> All();
        Cargo Find(int id);
        void Add(Cargo cargo);
        void Update(Cargo cargo);
        void Delete(int id);
    }
}

File: source/back-end/src/main/java/be/pxl/backend/service/ICargoService.java
package be.pxl.backend.service;

import java.util.List;
import be.pxl.backend.entity.Cargo;

public interface ICargoService {

	public Cargo find(int id);
	
	public List<Cargo> all();
	
	public void persist(Cargo cargo);
	
	public void delete(int id);
	
	public void update(Cargo cargo);
	
	public List<Cargo> getCargoBySensor(int sensor_id);

}
